homework/content.js

```diff
@@ -1,37 +1,73 @@
 'use strict';
 
-(function() {
-  var _wrapper = document.querySelector('#note-content-wrapper');
+(function(exports) {
+	
+	/**
+	 *Creates an instance of TodoContentManager.
+	 *
+	 *@constructor
+	 *@this {TodoContentManager}
+	 */
+    var TodoContentManager = function() {
+        /** @private */this._wrapper = null;
+    };
 
-  function start() {
-    window.addEventListener('note-open', function(event) {
-      var note = event.detail;
-      resetWrapper();
-      drawNote(note);
-    });
-  }
+	/**
+	 *Clear the Content.
+	 *
+	 *@this {TodoContentManager}
+	 */
+    TodoContentManager.prototype.resetWrapper = function() {
+        this._wrapper.innerHTML = '';
+    };
 
-  function resetWrapper() {
-    _wrapper.innerHTML = '';
-  }
+	/**
+	 *Write down the passage in the Content.
+	 *
+	 *@param {string} note is event.detail which is the passage.
+	 *@this {TodoContentManager}
+	 */
+    TodoContentManager.prototype.drawNote = function(note) {
+        var title = note.title;
+        var h = document.createElement('h2');
+        h.textContent = title;
+        var passages = note.passages;
+        var buff = document.createDocumentFragment();
+        passages.forEach(function(passage) {
+            var p = document.createElement('p');
+            p.classList.add('note-passage');
+            p.textContent = passage;
+            buff.appendChild(p);
+        });
+        this._wrapper.appendChild(h);
+        this._wrapper.appendChild(buff);
+    };
 
-  function drawNote(note) {
-    var title = note.title;
-    var h = document.createElement('h2');
-    h.textContent = title;
-    var passages = note.passages;
-    var buff = document.createDocumentFragment();
-    passages.forEach(function(passage) {
-      var p = document.createElement('p');
-      p.classList.add('note-passage');
-      p.textContent = passage;
-      buff.appendChild(p);
-    });
-    _wrapper.appendChild(h);
-    _wrapper.appendChild(buff);
-  }
+	/**
+	 *Handle the event.
+	 *
+	 *@param {object} Only 'note-open' will happen here.
+	 *@this {TodoContentManager}
+	 */
+    TodoContentManager.prototype.handleEvent = function(event) {
+        switch (event.type) {
+            case 'note-open':
+                var note = event.detail;
+                this.resetWrapper.bind(this)();
```

Comment:
**WRONG**: if you call `addEventListener` correctly, the `this` is already bound. Please check with that again, since code like this just show you didn't understand what I've taught in the class.